packages/frontend/src/hooks/echarts/useEcharts.ts

```diff
@@ -1435,13 +1439,16 @@ const useEcharts = (validCartesianConfigLegend?: LegendValues) => {
             color: colors,
         }),
         [
-            axis,
-            colors,
-            series,
+            axis.xAxis,
+            axis.yAxis,
             stackedSeries,
-            validCartesianConfig,
-            sortedResults,
+            isInDashboard,
+            validCartesianConfig?.eChartsConfig.legend,
+            validCartesianConfig?.eChartsConfig.grid,
             validCartesianConfigLegend,
+            series,
+            sortedResults,
+            colors,
```

Comment:
there is one more config for pie charts. it's called `useEchartsPieConfig`

Reply:
niceee, thank you! will do